### ISSUE TITLE
[Feature request] Update import paths on folder rename or move

### ISSUE THREAD

**Issue by olafkruger** [BowlerHatLLC/vscode-nextgenas#254]
Hi,
today, we have to update all imports of affected files manually after renaming or moving a folder.
(Or do I miss something?)
It would be very helpful if VSCode would do that automatically ;-)

There were already filed a couple of issues regarding this feature, maybe this helps [1].

Thanks,
Olaf

[1] https://github.com/Microsoft/vscode/issues/53233

**Comment by angelhdz** [comment]
Yup, I come from **FlashDevelop** and was expecting all the references to change with class file rename. I hope it gets added soon because having to navigate .as file by .as file and changing all references is time consuming and a pain in the **trace( ["a", "r", "s", "e"].join("") ).**

Thank you!

**Comment by joshtynjala** [comment]
@angelhdz  If you're only renaming a single class, you should be able to use **Rename Symbol** instead of renaming the file in the file explorer, and it will also change all references. You can find Rename Symbol when you right click in the editor.

**Comment by varadig** [comment]
What about this feature? It's would be really helpful
